src/taskprovider.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as vscode from 'vscode';
-import { isString } from 'util';
+import {isString} from './util';
```

Comment:
Saw this while searching for `for..in` usages in the codebase so I just bundled it in.  We were using the wrong `isString`.